store/postgres/src/subgraph_store.rs

```diff
@@ -676,11 +676,11 @@ impl SubgraphStoreInner {
         for_subscription: bool,
     ) -> Result<(Arc<DeploymentStore>, Arc<Site>, ReplicaId), StoreError> {
         let id = match target {
-            QueryTarget::Name(name) => self.mirror.current_deployment_for_subgraph(&name)?,
-            QueryTarget::Deployment(id) => id,
+            QueryTarget::Name(name, _) => self.mirror.current_deployment_for_subgraph(&name)?,
+            QueryTarget::Deployment(id, _) => id,
         };
 
-        let (store, site) = self.store(&id)?;
+        let (store, site) = self.store(&id)?; // TODO: maybe use a tuple (id, version) here?
```

Comment:
- [ ] not sure if we want to get the store based on `id` or `(id, version)`.

Reply:
Since the version only affects the API schema, I don't think there's a point in having the version here. I think version should only come into play for the `QueryStore`